lte/gateway/c/core/oai/tasks/grpc_service/AmfServiceImpl.h

```diff
@@ -51,6 +51,12 @@ class AmfServiceImpl final : public SmfPduSessionSmContext::Service {
   grpc::Status SetSmfSessionContext(ServerContext* context,
                                     const SetSMSessionContextAccess* request,
                                     SmContextVoid* response) override;
+  bool fillUpPacketFilterContents(packet_filter_contents_t* pf_content,
```

Comment:
These funs() , fillUpPacketFilterContents(), fillIpv6() and fillIpv4() are already defined in tasks/grpc_service/SpgwServiceImpl.cpp. Put in common module and use in both modules